lib/Resque.php

```diff
@@ -132,72 +138,9 @@ public static function pop($queue)
 	}
 
 	/**
-	 * Remove items of the specified queue
-	 *
-	 * @param string $queue The name of the queue to fetch an item from.
-	 * @param array $items
-	 * @return integer number of deleted items
-	 */
-	public static function dequeue($queue, $items = Array())
-	{
-	    if(count($items) > 0) {
-		return self::removeItems($queue, $items);
-	    } else {
-		return self::removeList($queue);
-	    }
-	}
-
-	/**
-	 * Remove specified queue
-	 *
-	 * @param string $queue The name of the queue to remove.
-	 * @return integer Number of deleted items
-	 */
-	public static function removeQueue($queue)
-	{
-	    $num = self::removeList($queue);
-	    self::redis()->srem('queues', $queue);
-	    return $num;
-	}
-
-	/**
-	 * Pop an item off the end of the specified queues, using blocking list pop,
-	 * decode it and return it.
-	 *
-	 * @param array         $queues
-	 * @param int           $timeout
-	 * @return null|array   Decoded item from the queue.
-	 */
-	public static function blpop(array $queues, $timeout)
-	{
-	    $list = array();
-	    foreach($queues AS $queue) {
-		$list[] = 'queue:' . $queue;
-	    }
-
-	    $item = self::redis()->blpop($list, (int)$timeout);
-
-	    if(!$item) {
-		return;
-	    }
-
-	    /**
-	     * Normally the Resque_Redis class returns queue names without the prefix
-	     * But the blpop is a bit different. It returns the name as prefix:queue:name
-	     * So we need to strip off the prefix:queue: part
-	     */
-	    $queue = substr($item[0], strlen(self::redis()->getPrefix() . 'queue:'));
-
-	    return array(
-		'queue'   => $queue,
-		'payload' => json_decode($item[1], true)
-	    );
-	}
```

Comment:
Blocking pop is *considerably* more performant and less resource-intensive than the older polling-pop.  It also needs to stay.